activemodel/lib/active_model/type/date_time.rb

```diff
@@ -39,9 +39,9 @@ def fallback_string_to_time(string)
         end
 
         def value_from_multiparameter_assignment(values_hash)
-          missing_parameter = (1..3).detect { |key| !values_hash.key?(key) }
-          if missing_parameter
-            raise ArgumentError, missing_parameter
+          missing_parameters = (1..3).select { |key| !values_hash.key?(key) }
+          if missing_parameters.present?
```

Comment:
`missing_parameters` is always an array instance, so it is enough using `Enumerable#any?` or `Array#empty?` over AS's `present?` method.